.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+<!--
+* New contributors are highly encouraged to read our
+  [CONTRIBUTING](/CONTRIBUTING.md) documentation.
+* Commit and PR titles should be prefixed with the general area of the pull request's change.
+
+-->
+### What does this PR do?
+
+<!--
+* A brief description of the change being made with this pull request.
+* If the description here cannot be expressed in a succinct form, consider
+  opening multiple pull requests instead of a single one.
+-->
+
+### Motivation
+
+<!--
+* What inspired you to submit this pull request?
+* Link any related GitHub issues or PRs here.
+-->
+
+### Additional Notes
```

Comment:
I personally feel that the datadog-agent template is a little too verbose (we basically never use all of the sections, which tells me there are too many), so I'd remove the "Additional Notes" and the "Possible Drawbacks / Trace-offs" sections. and just let that be in the "What does this PR do?" section